apps/ui/src/helpers/turbo.ts

```diff
@@ -0,0 +1,22 @@
+export const MAX_BODY_LENGTH = {
+  default: 10000,
+  turbo: 40000
+};
+
+export const MAX_CHOICES = {
+  default: 500,
```

Comment:
Can you change default to be 10 for now?

Reply:
Why? to test?
try running this in the console

```javascript
for (let i = 0; i < 500; i++) {
  document.querySelector(".static .s-base.mb-5 button.rounded-full.border.button.text-skin-link.bg-skin-bg.w-full.flex.items-center.justify-center.space-x-1").click();
}
```